nixos/modules/services/web-apps/cloudlog.nix

```diff
@@ -63,7 +63,7 @@ let
       ln -s ${dbFile} $out/application/config/database.php
 
       # link writable directories
-      for directory in updates uploads backup logbook; do
+      for directory in updates uploads backup logbook assets/qslcard images/eqsl_card_images; do
```

Comment:
I've only fixed features I use and could validate, but looking at the [installation docs](https://github.com/magicbug/Cloudlog/wiki/Installation#3-set-directory-ownership-and-permissions) I'm quite sure we also need to add `assets/sstvimages`

Reply:
Looking at https://github.com/magicbug/Cloudlog/pull/3031 I can confirm that we also need `assets/sstvimages`.